packages/shaku-code-annotate/src/index.ts

```diff
@@ -211,23 +237,17 @@ type ShakuComponent = ShakuComponentCallout | ShakuComponentUnderline;
 export function renderComponent(component: ShakuComponent) {
   switch (component.type) {
     case "ShakuComponentCallout":
-      return `<div class="shaku-callout" style="left:${
-        component.config.offset
-      }ch"><span class="shaku-callout-arrow" style="left:${
-        component.config.arrowOffset
-      }ch"></span>${component.config.contents
-        .map(escapeHtml)
-        .join("\n")}</div>`;
+      return `<div class="shaku-callout" style="left:${component.config.offset
```

Comment:
what is the cause of these changes?

please run `npm run format` in the root dir

Reply:
fix it

Reply:
Forgot to format the code before submitting. Thanks for catching that!